docs/source/alignprop_trainer.mdx

```diff
@@ -0,0 +1,117 @@
+# Aligning Text-to-Image Diffusion Models with Reward Backpropagation
+
+## The why
```

Comment:
I don't think as a reader I understand if the following table justifies the name of this section. Would you mind elaborating?

Reply:
Thanks,  I added a better why statement.